data/simple-icons.json

```
@@ -5821,6 +5821,11 @@
 		"source": "https://fritz.com",
 		"guidelines": "https://business.avm.de/de/data/allgemeine-nutzungsbedingungen-business-portal"
 	},
+	{
+		"title": "Fronius",
+		"hex": "D73237",
+		"source": "https://fronius.com"
```

```suggestion
		"source": "https://www.fronius.com/-/media/system/favicons/favicon-fronius-svg.svg"
```

Icon looks prefect, don't forget change source.